src/ray/stats/metric_defs.cc

```diff
@@ -114,9 +116,7 @@ DEFINE_stats(
     (),
     ray::stats::GAUGE);
 
-double operator""_MB(unsigned long long int x) {
-  return static_cast<double>(1024L * 1024L * x);
-}
+double operator""_MB(unsigned long long int x) { return static_cast<double>(x * 1_MiB); }
```

Comment:
can we just remove this and directly use your _MiB ?

Reply:
Updated -- was resolving the compilation issue caused by `size_t`.